R/extremes.R

```diff
@@ -22,13 +22,20 @@ setMethod("extremes", signature = c("list", "character"),
           }
 )
 
+
 #'@importFrom httr GET
-setMethod("extremes", signature = c("character", "character"), 
-          definition = function(data, output,...) {
-            cat('this function authenticates, goes out and gets data, and then calls the other one\n')
-            
-            token <- paste0('Bearer ', authenticateUser(...))
-            ts_list <- getJSON(url = data, auth = token)
+setMethod("extremes", signature = c("character", "character", "character"), 
+          definition = function(data, output, token) {
+            cat('this function uses the token to get data, and then calls the other one\n')
+           
+            authHeader <- paste("Bearer" , token)
```

Comment:
need to use `paste0` hear, as the result of `paste('a','b')` is `[1] 'a', 'b'` not `'ab'`